tests/users/test_api_list_users.py

```diff
@@ -50,6 +50,19 @@ def test_list_users_api_resource_auth(self):
         self.assertEqual(200, actual_response.status_code)
         self.assertEqual(expected_response, json.loads(actual_response.data))
 
+    def test_search_user_api_resource_auth(self):
+        auth_header = get_test_request_header(self.admin_user.id)
+        print("verified user: %s"%self.verified_user.name)
+        print("other user: %s"%self.other_user.name)
+        expected_response = marshal(self.verified_user, public_user_api_model)
+        actual_response = self.client.get(
+            '/users',data = {'search': self.verified_user.name},
```

Comment:
Can we search partial as well?

Reply:
Yes Anna. In the attached screenshot I searched for term "gci". This gave various names containing the term like gcitaskthree, gciuserfive and so on.